src/modules/components/asset/assetTransfer/assetTransfer.tsx

```diff
@@ -44,29 +44,10 @@ export interface IAssetTransferProps extends IWeb3ComponentProps {
 }
 
 export const AssetTransfer: React.FC<IAssetTransferProps> = (props) => {
-    const {
-        sender,
-        recipient,
-        assetName,
-        assetIconSrc,
-        assetAmount,
-        assetSymbol,
-        assetFiatPrice,
-        chainId,
-        hash,
-        wagmiConfig: wagmiConfigProps,
-    } = props;
+    const { sender, recipient, assetName, assetIconSrc, assetAmount, assetSymbol, assetFiatPrice, chainId, hash } =
+        props;
 
-    const wagmiConfigProvider = useConfig();
-
-    const wagmiConfig = wagmiConfigProps ?? wagmiConfigProvider;
-
```

Comment:
Looks like this logic got lost, the chain-id for this component follows this priority:
```
1. chainId component property
2. Custom wagmiConfig component property
3. Global wagmiConfig property coming from the provider
```
Maybe we could move part of this logic to the `useBlockExplorer` hook and share it?